src/contracts/SafientMain.ts

```diff
@@ -96,11 +96,11 @@ export class SafientMain {
    * @param evidenceURI - IPFS URI pointing to the evidence submitted by the claim creator
    * @returns A transaction response
    */
-  createClaim = async (safeIdOnThreadDB: string, evidenceURI: string): Promise<TransactionResponse> => {
+  createClaim = async (safeIdOnThreadDB: string, evidenceURI: string): Promise<any> => {
```

Comment:
Can we not use `TransactionResponse` type here?

Reply:
Oh, yeah sorry forgot that. Will change that